frontend/app/config/webpack.production.js

```diff
@@ -46,7 +46,7 @@ const config = {
       PUBLIC_URL,
     }),
     new Dotenv({
-      path: commonPaths.prodEnv,
+      systemvars: true,
```

Comment:
Just so to now, what does this line do?

Reply:
that plugin filters the env variables for default, if you set systemvars true, you allow variables defined in other parts of the system to be read, for example, the ones you set in amplify

Reply:
cool! thenks!!